src/rust/bitbox02-rust/src/workflow/mnemonic.rs

```diff
@@ -13,26 +13,36 @@
 // limitations under the License.
 
 pub use super::cancel::Error as CancelError;
-use super::cancel::{cancel, set_result, with_cancel};
 use super::confirm;
-use super::menu;
-use super::status::status;
-use super::trinary_choice::{choose, TrinaryChoice};
-use super::trinary_input_string;
 
-use alloc::boxed::Box;
 use alloc::string::String;
-use alloc::vec::Vec;
-use core::cell::RefCell;
 
-use sha2::{Digest, Sha256};
+cfg_if::cfg_if! {
```

Comment:
i think with that many changes, it's better to add a new file mnemonic_stub_c_unit_tests.rs, so the real firmware code does not become obfuscated by all this